src/components/Dashboard/ThisWeek/PartnerCare/PartnerCare.tsx

```diff
@@ -103,8 +154,29 @@ const PartnerCare = ({
     setValue(newValue);
   };
 
+  const handleCreateClick = (
+    celebrationType: CelebrationTypeEnum,
+    person: GetThisWeekQuery['reportsPeopleWithAnniversaries']['periods'][0]['people'][0] &
+      GetThisWeekQuery['reportsPeopleWithBirthdays']['periods'][0]['people'][0],
+  ) => {
+    openTaskDrawer({
+      defaultValues: {
+        subject:
+          celebrationType === CelebrationTypeEnum.birthday
+            ? `${person.firstName} ${person.lastName}'s Birthday`
+            : `${person.parentContact.name}'s Anniversary`,
```

Comment:
Let me know what you think of this. It doesn't currently do this in MPDx, but it seemed easy enough to add.